packages/testing-tachyons/src/Css.test.tsx

```diff
@@ -123,8 +123,58 @@ describe("Css", () => {
     Css.black.if("rawstring").$;
   });
 
-  it("cannot 'else' when using `if(bp)`", () => {
-    expect(() => Css.ifSm.black.else.white.$).toThrow("else is not supported");
+  it("can use 'else' when using `ifSm`", () => {
+    expect(Css.ifSm.black.else.white.$).toMatchInlineSnapshot(`
+      {
+        "@media not screen and (max-width:599px)": {
+          "color": "#fcfcfa",
+        },
+        "@media screen and (max-width:599px)": {
+          "color": "#353535",
+        },
+      }
+    `);
+  });
+
+  it("can use 'else' when using `ifMdAndUp`", () => {
+    expect(Css.ifMdAndUp.black.else.white.$).toMatchInlineSnapshot(`
+      {
+        "@media not screen and (min-width:600px)": {
+          "color": "#fcfcfa",
+        },
+        "@media screen and (min-width:600px)": {
+          "color": "#353535",
+        },
+      }
+    `);
+  });
+
+  it("cannot use 'else' twice when using `ifMdAndUp`", () => {
+    expect(() => Css.ifMdAndUp.black.else.white.else.blue.$).toThrowError("else was already called");
+  });
+
+  it.skip("can use 'else' twice for conditional then breakpoint use 'else' twice when using `ifMdAndUp`", () => {
```

Comment:
Leaving this support, of like mixing both `if` and breakpoint conditionals, to a future PR.